test_0.py

```diff
@@ -0,0 +1,10 @@
+def qwe(a, b, c):
+    print(a)
+
+
+def w(a: str) -> str:
+    return a
+
+
+def test_w():
+    assert w(2) == 2
```

Comment:
xDDDDDDDDDD

Why doesn't mypy fail?

Reply:
I have no idea, but it's funny that GH allows you to merge pull request before resolving comments :D